scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -554,9 +557,19 @@ def _publish(self) -> Dict[str, object]:
         return self._yaml.get('publish')
 
     @property
-    def description(self) -> Dict[str, object]:
+    def _description_file_path(self) -> Path:

```

Comment:
Validation would fail before this returns a `None`. The function could just be simplified to `return self._file_path / self._publish.get('description')`

Reply:
Updated